webapp/src/main/webapp/.gitignore

```diff
@@ -1,2 +1,3 @@
 META-INF/maven
-META-INF/MANIFEST.MF
\ No newline at end of file
+META-INF/MANIFEST.MF
+WEB-INF/classes/META-INF/services/javax.xml.stream.XMLInputFactory
```

Comment:
This file does not exist in this location

Reply:
After running maven war:inplace it exists (coming from the iaf-war overlay, it's in the classes folder of the webapp war)

Reply:
kan je `javax.xml.stream.XMLInputFactory` vervangen door `/**`

Reply:
Done